electron/src/lib/LocalAccountDeletion.ts

```diff
@@ -77,8 +78,13 @@ export async function deleteAccount(id: number, accountId: string, partitionId?:
       throw new Error('Account is not an UUID');
     }
     const sessionFolder = path.join(LOG_DIR, accountId);
+    const relativeFilePaths = getLogFilenames();
     await fs.remove(sessionFolder);
-    logger.log(`Deleted logs folder for account "${accountId}".`);
+    relativeFilePaths.forEach(async log => {
+      const logPath = path.join(LOG_DIR, log);
+      await fs.remove(logPath);
+    });
+    logger.log(`Deleted logs folder for current account.`);
   } catch (error: any) {
     logger.error(`Failed to delete logs folder for account "${accountId}", reason: "${error.message}".`);
```

Comment:
Shouldn't we change that as well?

Reply:
In this case I guess logs probably weren't deleted so I thought good (or acceptable) to leave it here? Idk I'm open either way

Reply:
I'd be open to leave an extended warning here to tell the user to manually delete their logs, with the location. But idk if that's best

Reply:
Thing is, this log will go to the main `electron.log` isn't that correct. I guess we should never put any account ID in those logs (that's my understanding)

Reply:
So I can imagine that in some cases it makes sense to have userId logs in this file. But just to clear them when the user is deleted. But the ticket does indeed say "don't log userId"

Reply:
wait, are you removing the entire `electron.log` and `electron.old` in this PR? Might be a little overkill 🤔 
We are going to lose logs for other accounts